src/actionTypes.js

```diff
@@ -344,6 +344,9 @@ type EventUpdateMessageAction = {|
   type: typeof EVENT_UPDATE_MESSAGE,
   edit_timestamp: number,
   message_id: number,
+  message_ids: number[],
```

Comment:
I just mentioned linking to the API docs, and I had the thought that that substitutes for writing down any documentation here about what the properties mean... but as I read this event's docs, I see that this `message_id` vs. `message_ids` thing is odd enough, and enough of a potential gotcha, that it probably merits making an exception and writing something down here.

In particular, any content changes apply to just `message_id`, and any stream or topic changes apply to all of `message_ids`, which is guaranteed to include `message_id`.

Reply:
The docs also kind of make it sound like `message_ids` isn't always present -- emphasis added:
> *If the stream or topic was changed*, the set of moved messages is encoded in the separate message_ids field, which is guaranteed to include message_id.

even though the `message_ids` bit doesn't say it's optional. That might be good to clarify in chat on #api-design.

Reply:
Confirmed in [chat](https://chat.zulip.org/#narrow/stream/378-api-design/topic/.60update_message.60.20event/near/1293909) that we do expect it to be always present.